src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/support/SupportController.java

```diff
@@ -98,6 +106,42 @@ public void resendStaffNotifications(
         }
     }
 
+    @PutMapping("/claim/{referenceNumber}/claimDocumentType/{claimDocumentType}/uploadDocumentToDocumentManagement")
+    @ApiOperation("Upload document to document Management")
+    public void uploadDocumentToDocumentManagement(
+        @PathVariable("referenceNumber") String referenceNumber,
+        @PathVariable("claimDocumentType") ClaimDocumentType claimDocumentType,
+        @RequestHeader(value = HttpHeaders.AUTHORIZATION, required = false) String authorisation) {
```

Comment:
Hope it should be true. Please correct or give reason if anything specific to requirement